.dassie/app/models/collection_resource.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Generated via
+#  `rails generate hyrax:collection_resource CollectionResource`
+class CollectionResource < Hyrax::PcdmCollection
+  # @note Do not directly update `basic_metadata.yaml`.  It is also used by works.
+  #
+  # To change metadata for collections
+  # * extend by adding fields to `/config/metadata/CollectionResource_metadata.yaml`
+  # * remove all or some basic metadata fields
+  #   * comment out or delete the schema include statement for `:basic_metadata`
+  #   * modify the contents of `/config/metadata/CollectionResource_metadata.yaml`
```

Comment:
Another small place to make dassie consistent with the generator:
```suggestion
  # * extend by adding fields to `/config/metadata/collection_resource.yaml`
  # * remove all or some basic metadata fields
  #   * comment out or delete the schema include statement for `:basic_metadata`
  #   * modify the contents of `/config/metadata/collection_resource.yaml`
```